simd-minimizers/src/test.rs

```diff
@@ -249,3 +249,164 @@ fn canonical_minimizer_positions() {
         assert_eq!(scalar_ascii, simd_packed, "k={k}, w={w}, len={len}");
     });
 }
+
+#[test]
+fn minimizer_and_superkmer_positions() {
+    test_on_inputs(|k, w, _slice, ascii_seq, packed_seq| {
+        let mut scalar_ascii = vec![];
```

Comment:
trick to possibly make things cleaner: `let scalar_ascii = &mut vec![];`, and then drop the `&mut` later one. May fold things into one line.